manager/controllers/advisory_systems_test.go

```diff
@@ -178,3 +178,29 @@ func TestAdvisorySystemsTagsUnknown(t *testing.T) { //nolint:dupl
 func TestAdvisorySystemsWrongOffset(t *testing.T) {
 	doTestWrongOffset(t, "/:advisory_id", "/RH-1?offset=1000", AdvisorySystemsListHandler)
 }
+
+func TestAdvisorySystemsSortByOsName(t *testing.T) { //nolint:dupl
+	utils.SkipWithoutDB(t)
+	core.SetupTestEnvironment()
+
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", "/RH-1?sort=os_name", nil)
+	core.InitRouterWithPath(AdvisorySystemsListHandler, "/:advisory_id").ServeHTTP(w, req)
+
+	var output AdvisorySystemsResponse
+	ParseReponseBody(t, w.Body.Bytes(), &output)
+
+	assert.Equal(t, 6, len(output.Data))
+	assert.Equal(t, "00000000-0000-0000-0000-000000000001", output.Data[0].ID)
+	assert.Equal(t, "RHEL", output.Data[0].Attributes.OSName)
+	assert.Equal(t, "8", output.Data[0].Attributes.OSMajor)
```

Comment:
These tests verify only sorting according to `os_minor`. Btw. currently the sorting fields order is wrong (os_name, os_minor, os_major). The field os_major should have higher priority, shouldn't it?